SpatialGDK/Extras/schema/spawner.schema

```diff
@@ -6,6 +6,7 @@ type SpawnPlayerRequest {
     bytes unique_id = 2;
     string online_platform_name = 3;
     bool simulated = 4;
+    EntityId client_system_entity_id = 5;
```

Comment:
We add this because we need the server to about client worker system entity since it's necessary to assign the client worker authority over UnrealClientEndpoint and Heartbeat components with partitions

Reply:
Shouldn't this be called `client_partition_id` ?

Reply:
Oh I see these are actually two different things.
And the reason we use the pc entity id instead of the client's system entity id as the partition id is because the system entity would be automatically deleted/refreshed if the client disconnected/rejoined?

Reply:
Rejoining was one issue with this. ID reuse over snapshots was another potential problem (I can't remember if this was resolved or not).

Reply:
Don't know enough about the rejoining flow, assuming the PlayerController wouldn't be maintained so creation of a new one would be fine here?